lib/transaction/sidechain.js

```diff
@@ -1,20 +1,18 @@
 'use strict';
 
-var _ = require('lodash');
-var $ = require('../util/preconditions');
-var BN = require('../crypto/bn');
-var Hash = require('../crypto/hash');
-var buffer = require('buffer');
-var BufferWriter = require('../encoding/bufferwriter');
-var BufferReader = require('../encoding/bufferreader');
-var BufferUtil = require('../util/buffer');
-var JSUtil = require('../util/js');
+const _ = require('lodash');
+const $ = require('../util/preconditions');
+const BN = require('../crypto/bn');
+const buffer = require('buffer');
+const BufferWriter = require('../encoding/bufferwriter');
+const BufferUtil = require('../util/buffer');
+const JSUtil = require('../util/js');
 
 function Sidechain(params) {
```

Comment:
Do we want to add doc description for this class?

Reply:
Same